visualization/.eslintrc.js

```diff
@@ -14,48 +20,24 @@ module.exports = {
 		"@typescript-eslint"
 	],
 	"rules": {
-		"@typescript-eslint/class-name-casing": "error",
-		"@typescript-eslint/explicit-member-accessibility": [
-			"error",
-			{
-				"accessibility": "explicit",
-				"overrides": {
-					"constructors": "no-public"
-				}
-			}
-		],
-		"@typescript-eslint/triple-slash-reference": "error",
-		"curly": "error",
-		"new-parens": "error",
-		"no-caller": "error",
-		"no-cond-assign": "error",
-		"no-console": [
-			"warn",
-			{
-				"allow": [
-					"warn",
-					"dir",
-					"timeLog",
-					"assert",
-					"clear",
-					"count",
-					"countReset",
-					"group",
-					"groupEnd",
-					"table",
-					"dirxml",
-					"error",
-					"groupCollapsed",
-					"Console",
-					"profile",
-					"profileEnd",
-					"timeStamp",
-					"context"
-				]
-			}
-		],
-		"no-empty": "error",
-		"no-eval": "error",
-		"no-var": "error"
+		// Do not apply inappropriate rules below
+		"@typescript-eslint/no-inferrable-types": "off",
+		"@typescript-eslint/no-use-before-define": "off",
+
+		// TODO fix and remove rules below
+		"@typescript-eslint/ban-ts-ignore": "off",
```

Comment:
We use ts-ignore in the injector services.

Reply:
That is true..

Reply:
Should disable that rule then or show warnings instead if errors?

Reply:
disable it

Reply:
done